src/coreclr/src/vm/method.cpp

```diff
@@ -5334,10 +5334,14 @@ LPVOID NDirectMethodDesc::FindEntryPoint(NATIVE_LIBRARY_HANDLE hMod) const
     }
 #endif
 
-    // Just look for the user-provided name without charset suffixes.  If it is unicode fcn, we are going
+    // Just look for the user-provided name without charset suffixes.
+    // If we are targetting Windows and it is unicode fcn, we are going
     // to need to check for the 'W' API because it takes precedence over the
     // unmangled one (on NT some APIs have unmangled ANSI exports).
     pFunc = FindEntryPointWithMangling(hMod, funcName);
```

Comment:
Can we also strip the Windows-specific entrypoint mangling and just call `PAL_GetProcAddressDirect` on Unix?

Reply:
Since we don't need to worry about the stdcall mangling on Unix x64 (and x86 is unsupported), sure! I'll go make the change.

Reply:
Unix x86 does not use the mangling anyway.

Reply:
Didn't know that. Awesome!